Sources/ArgumentParser/Parsing/InputOrigin.swift

```diff
@@ -109,15 +113,15 @@ extension InputOrigin {
   }
 }
 
-extension InputOrigin.Element {
```

Comment:
If this is still compiling, this seems like leftover code that we aren't using anymore. Feel free to delete!

Reply:
done